Source/WebKit/NetworkProcess/storage/CacheStorageCache.cpp

```diff
@@ -136,6 +136,8 @@ static CacheStorageRecord toCacheStorageRecord(WebCore::DOMCacheEngine::CrossThr
     NetworkCache::Key key { "record"_s, uniqueName, { }, createVersion4UUIDString(), salt };
     CacheStorageRecordInformation recordInfo { WTFMove(key), MonotonicTime::now().secondsSinceEpoch().milliseconds(), record.identifier, 0 , record.responseBodySize, record.request.url(), false, { } };
     recordInfo.updateVaryHeaders(record.request, record.response.httpHeaderFields.get(WebCore::HTTPHeaderName::Vary));
+    if (record.response.tainting == WebCore::ResourceResponse::Tainting::Opaque || record.response.tainting == WebCore::ResourceResponse::Tainting::Opaqueredirect)
+        recordInfo.varyHeaders = { };
```

Comment:
Do we need to do the same thing (clearing varyHeaders) when reading record from disk, i.e. in `readRecordInfoFromFileData`?

Reply:
Good point, I am changing the place of the check